tests/roles/ovn_adoption/tasks/main.yaml

```diff
@@ -121,7 +121,6 @@
         enabled: true
         template:
           ovnNorthd:
-            containerImage: {{ container_registry|default("quay.io") }}/{{ container_namespace|default("podified-antelope-centos9") }}/openstack-ovn-northd:{{ container_tag | default("current-podified") }}
```

Comment:
Good catch i'll add it.

Reply:
Done. I also fixed indentation where i noticed it being off in that file.

Reply:
right - I left these where they were and meant to cycle back and check if we really wanted/needed them hard-coded here.

thanks @jistr adding to the overrides should keep the period jobs doing the right thing, and otherwise it will default to current-podified